### PR TITLE
[READY] - Adding cacert pkg to awsutils img

### PR DESCRIPTION
**Pull request by sarcasticadmin**
`curl` isnt very useful if you dont have a certificate authority. Got the follow error due to no ca trusts being present:

```
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

### CODE REVIEW

**Comment by sarcasticadmin** [comment]
publish

File: imgs/awsutils/default.nix [hunk 1]
```diff
@@ -3,7 +3,7 @@ let
   nwi = import ../../nwi.nix;
   pkgs = import ../../pin { snapshot = "nixos-20-03_0"; };
   lib = pkgs.lib;
-  contents = [ pkgs.coreutils pkgs.bash pkgs.jq pkgs.curl pkgs.awscli ];
+  contents = with pkgs; [ cacert coreutils bash jq curl awscli ];
```
Comment (maishiroma) [inline imgs/awsutils/default.nix:6]: Good call on cleaning up the syntax here, much easier to infer where each package is coming from.

**Comment by maishiroma** [comment]
publish

**Comment by maishiroma** [comment]
Looks like there's an issue with publishing one of the images onto Docker Hub if you look at the logs in the GitHub Actions for publishing images:
```
...
Finished.
/nix/store/pbj9q2qyw511wfy18g8lz9jw2mpc5qvw-docker-image-helmsman-aws.tar.gz
+ cp /__w/nix-garage/nix-garage/imgs/helmsman-aws/result /__w/nix-garage/nix-garage/result
+ nix-shell --run './publish-imgs helmsman-aws'
warning: you did not specify '--add-root'; the result might be removed by the garbage collector
time="2020-06-30T15:11:46Z" level=fatal msg="Error parsing image name \"docker://docker.io/nebulaworks/helmsman-aws:pbj9q2qyw511wfy18g8lz9jw2mpc5qvw\": Error reading manifest pbj9q2qyw511wfy18g8lz9jw2mpc5qvw in docker.io/nebulaworks/helmsman-aws: manifest unknown: manifest unknown"
Getting image source signatures
Copying blob sha256:4864e02891e4b109b1a8399507183f666a1a04340ac51a4c7203281836254d14
time="2020-06-30T15:11:49Z" level=fatal msg="Error writing blob: Error initiating layer upload to /v2/nebulaworks/helmsman-aws/blobs/uploads/ in registry-1.docker.io: errors:\ndenied: requested access to the resource is denied\nunauthorized: authentication required\n"
```

Not sure what the root cause is; this was something that was working in an earlier run. This may be something we'll have to investigate later. Interestingly enough, the image for `awsutil` went through fine. I can take a look at the action later and potentially create another issue to help make it more clear when the publishing step fails.

**Comment by maishiroma** [comment]
A fix for the workflow issue is at #12 , so once we can verify that fix in here, we can go ahead and get this moving

**Comment by sarcasticadmin** [comment]
publish

**Comment by sarcasticadmin** [comment]
publish

**Comment by sarcasticadmin** [comment]
publish

**Comment by nwi-auto** [comment]
The following docker images: 
awsutils, helmsman-aws
have been published on [Docker Hub](https://hub.docker.com/u/nebulaworks).

**Comment by sarcasticadmin** [comment]
publish

**Comment by nwi-auto** [comment]
The following docker images: 
awsutils, helmsman-aws
have been published on [Docker Hub](https://hub.docker.com/u/nebulaworks).

**Comment by sarcasticadmin** [comment]
Adding some context around why helmsman-aws change. @maishiroma dug into why the derivation changed due to #10  and his findings were:

```
I just did a build of the old helmsman-aws image prior to the fmt and I was able to get the existing image hash. The change did
arose from the fmt, but it was in the deps.nix in the pkgs/helm-diff . By fmt that, it caused the hash to change in the 
nix-build, creating a new image for helmsman-aws. The odd thing though is that there is no heredoc that is being modified in
that nix config. In fact, the heredoc  doesn’t seem to matter in this case. awsutils has a heredoc that got modified in the
default.nix , but that was not enough of a difference to generate a brand new image (looking at the last job for publishing
images showed this).
```

Output taken from: https://github.com/Nebulaworks/nix-garage/runs/828756395?check_suite_focus=true
```
/nix/store/pbj9q2qyw511wfy18g8lz9jw2mpc5qvw-docker-image-helmsman-aws.tar.gz
+ cp /__w/nix-garage/nix-garage/imgs/helmsman-aws//result /__w/nix-garage/nix-garage/result
+ nix-shell --run './publish-imgs helmsman-aws'
warning: you did not specify '--add-root'; the result might be removed by the garbage collector
time="2020-07-02T00:06:45Z" level=fatal msg="Error parsing image name \"docker://docker.io/nebulaworks/helmsman-aws:pbj9q2qyw511wfy18g8lz9jw2mpc5qvw\": Error reading manifest pbj9q2qyw511wfy18g8lz9jw2mpc5qvw in docker.io/nebulaworks/helmsman-aws: manifest unknown: manifest unknown"
Getting image source signatures
Copying blob sha256:4864e02891e4b109b1a8399507183f666a1a04340ac51a4c7203281836254d14
Copying config sha256:252c03ebc4b2f7c4bd79baa8488a6d4a96d1ca9414ff2d4a4765b6701291b8cb
Writing manifest to image destination
Storing signatures
```

**Comment by sarcasticadmin** [comment]
@maishiroma thanks for all the help on this, this is now ready to go! 🚀

**Comment by maishiroma** [comment]
Awesome, thanks for putting in that verbose message on the changes from `helmsman-aws`! LGTM

**Pull request merged**